website/content/docs/commands/plugin/list.mdx

```diff
@@ -9,6 +9,12 @@ description: The "plugin list" command lists all available plugins in the plugin
 The `plugin list` command lists all available plugins in the plugin catalog.
 It can be used alone or with a type such as "auth", "database", or "secret".
 
+## Deprecation Status Column
+
+As of 1.12, all builtin plugins will have an associated Deprecation
+Status. This status will be reflected in the `Deprecation Status` column, seen
+below. All non-builtin plugins will show a `Deprecation Status` of "n/a".
```

Comment:
I find it a bit weird that non-builtin plugins will show a deprecation status of "n/a" here but the field in the plugin info API is omitted. Should it be omitted or "n/a" in both places?